### PR TITLE
Add Scraper for Atrius/Harvard Vanguard.

### PR DESCRIPTION
**Pull request by adpjay**
I named it Atrius instead of Harvard Vanguard as it is the site for all of Atrius health

### CODE REVIEW

File: site-scrapers/Atrius.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                response = JSON.parse(body);
+                resolve(response);
+            });
+        });
+        req.write(postData);
+        req.end();
+    });
+
+    const postResponse = await schedulePromise;
+    allDays = postResponse.AllDays
```
Comment (livgust) [inline site-scrapers/Atrius.js:98]: missing semicolon; I suggest installing "prettier" and using the default (I use 4 spaces for a tab - I'll put a config file in for this repo to make things more consistent)

File: site-scrapers/Atrius.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    const results = { availability: {}, hasAvailability: false };
+    // The AllDays property is an object, not an array.
+    // The property of the object is generated from the date, which we don't know
+    // We need to iterate over all the properties to determine the dates and the slots.
+    for (const dayEntryProperty in allDays) {
+        if (Object.prototype.hasOwnProperty.call(allDays, dayEntryProperty)) {
+            dayEntry = allDays[dayEntryProperty]
+            const date = new Date(dayEntry.DateISO);
+            let slots = dayEntry.Slots;
+            let slotsAvailable = slots?slots.length:0
```
Comment (livgust) [inline site-scrapers/Atrius.js:110]: spacing here is off; but actually can't you just do
```
slots.length || 0
```
?

Comment (livgust) [inline site-scrapers/Atrius.js:110]: both of these can be `const`s because they don't change.

**Review by livgust — CHANGES_REQUESTED** [review]
Looks good, just a few nitpicky comments! I haven't tested yet; I'll do that now. FYI you should install `prettier` to format your code so everything is consistent style-wise. LMK if you need assistance.

**Comment by livgust** [comment]
Also, ah! Your days are off by one! You need to add 1 to the date, maybe it's 0-indexed or something.
image online it shows 28 slots but your script is outputting 28 slots for the day before (which isn't even listed on the website)

**Comment by adpjay** [comment]
Good catches Liv. I'll fix those tonight.

File: site-scrapers/Atrius.js (hunk 1 above)
Comment (adpjay) [reply site-scrapers/Atrius.js]: Did you have a `prettier` config you wanted to use? I used the below. I'm happy to add that if it works.
```yml
# .prettierrc or .prettierrc.yaml
trailingComma: "es5"
tabWidth: 4
semi: true
singleQuote: true
```

Comment (livgust) [reply site-scrapers/Atrius.js]: DMed you on slack - that looks good except I think I've been using " throughout the repo so we'd want singleQuote false IIRC

**Review by livgust — APPROVED** [review]
LGTM! There's an extra space in sites.json (street address) and we're using double-quotes in the rest of the repo but I'll prioritize adding a prettierrc file and applying it to the repo on my own. Results from the scraper match what I see on the website.

**Pull request merged**